src/clients/api/queries/getTransactions/formatTransaction.ts

```diff
@@ -21,8 +19,6 @@ const formatTransaction = ({
   return {
     ...rest,
     amountWei: convertTokensToWei({ value: new BigNumber(amount), token }),
-    createdAt: new Date(createdAt),
```

Comment:
I would do this in 2 steps like you mentioned in slack.
First we can update the logic to use the timestamp and fallback to createdAt.
Once the backfill is complete we can remove createdAt and remove the fallback.

Reply:
I don't think we should fallback to the `createdAt` field, as we will display partially correct data. We technically display wrong data all the time currently, but I'd rather make only one update to the frontend that fixes the issue definitely.

The idea is that this PR would only be deployed once the backend has been updated to backfill all the `null` timestamp fields.

Reply:
Got it. Sounds good